### ISSUE TITLE
Option to disable the Trackers Preview

### ISSUE THREAD

**Issue by benoitballester** [ghostery/ghostery-extension#741]
### Description

An annoying "WhoTrackMe tracker wheel" appeared.
I want to remove it, but nothing in the settings seems to remove this annoying "WhoTrackMe tracker wheel". 

### Expected Behavior

no "WhoTrackMe tracker wheel" should be added to my search results. 

### Actual Behavior

An annoying "WhoTrackMe tracker wheel" appeared. I just hate when an extension messes with my search results. 

### Steps to Reproduce

Just get the latest update

### Versions

+ Browser: Chrome
+ OS: Mac OS 10.14
+ Node:
+ NPM:

**Comment by JacksonWrath** [comment]
Yes. This is a nice feature, but I personally do not care. There needs to be a toggle to disable this please!

**Comment by hm122** [comment]
Same. Really annoying when searching in google and all those wheels appear. Already thought about deinstalling it. Will certainly do if there is no fix.
<img width="137" alt="Screen Shot 2022-01-16 at 16 39 10" src="https://user-images.githubusercontent.com/13405226/149666823-f2c6dd1c-1e83-4750-87db-064dcda73161.png">

**Comment by dpupkov** [comment]
Completely agree, there should be a setting to turn the wheel off.

Workaround is to use any other AdBlock extension to remove the wheel. In my case, I'm using uBlock Origin to remove this annoying wheel from Ghostery. It sounds really stupid, but it works for me.

**Comment by chrmod** [comment]
We hear you, thank you for your feedback!

As we prepare a new extension release (Ghostery 8.6.1) with the option to disable the Tracker Wheel on Google, we’ll be writing a blog post to describe why we have added this feature in the first place. 

In short, its goal is to let you identify heavily tracked websites and be aware of this information before visiting them. 

We would love to hear more from you! If you have feedback or suggestions on how to improve this feature, we are all ears!

**Comment by nreimers** [comment]
Hi @chrmod 
thanks so much for your quick reaction, really appreciate your effort :). Looking forward to the updated version.

For me the WhoTracksMe does not provide value: In many cases there are just few relevant hits in the search results and what I will open will not depend on the track wheel, but if I think it is relevant from title + description (or it was the one specific site I was looking for - e.g. when I search for a certain Github repository, I don't care what the wheel shows). So it is currently just a distraction - especially as it came surprisingly and I didn't opted-in to have it. So I was really wondering what this is and where it comes from.

**Comment by chrmod** [comment]
The issue was automatically closed by a merged PR. Lets keep it open until 8.6.1 gets released to Chrome and Edge.

**Comment by philipp-classen** [comment]
8.6.1 has been released on Chrome and Opera. Unfortunately, releasing on Edge is slower:

* 13th of January: we pushed 8.6.0 (live version on Edge is 8.5.7)
* 17th of January: we pushed 8.6.1 (live version on Edge is still 8.5.7)
* 20th of January: Edge made the 8.6.0 release public

So, if you are on Edge and want to disable the Tracker Preview, the release with the option is waiting to be released. We would have preferred to see the updated version being released, but unfortunately we have no control over that. Apology, but the option will be available on Edge soon.

**Comment by philipp-classen** [comment]
8.6.1 is now live on Edge.

**Comment by philipp-classen** [comment]
Released now on all channels.

**Comment by chrmod** [comment]
Also in 8.6.2 we've added a quick disable button directly in the Trackers Preview so you don't have to look through settings.

<img width="452" alt="Screenshot 2022-02-11 at 14 53 40" src="https://user-images.githubusercontent.com/1228153/153603646-ee6b45ea-c3da-427f-9ceb-983b5d930e34.png">
